python/tank/authentication/defaults_manager.py

```diff
@@ -22,10 +21,62 @@ class DefaultsManager(object):
     on disk and the system maintains a concept of a current user and a current
     host.
 
+    The defaults manager will also look in multiple locations for a file called ``config.ini``
+    in order to provide appropriate defaults:
+
+        - The SGTK_CONFIG_LOCATION environment variable,
```

Comment:
I think we should split these locations into a list of the locations we want people to actually use today and then another list of supported legacy locations. Also need to update to support paths on all platforms, not just mac.